src/libraries/System.Linq.Queryable/ref/System.Linq.Queryable.cs

```diff
@@ -29,10 +29,10 @@ public partial class EnumerableQuery<T> : System.Linq.EnumerableQuery, System.Co
         System.Collections.Generic.IEnumerator<T> System.Collections.Generic.IEnumerable<T>.GetEnumerator() { throw null; }
         System.Collections.IEnumerator System.Collections.IEnumerable.GetEnumerator() { throw null; }
         System.Linq.IQueryable System.Linq.IQueryProvider.CreateQuery(System.Linq.Expressions.Expression expression) { throw null; }
-        System.Linq.IQueryable<S> System.Linq.IQueryProvider.CreateQuery<S>(System.Linq.Expressions.Expression expression) { throw null; }
+        System.Linq.IQueryable<TElement> System.Linq.IQueryProvider.CreateQuery<TElement>(System.Linq.Expressions.Expression expression) { throw null; }
         object System.Linq.IQueryProvider.Execute(System.Linq.Expressions.Expression expression) { throw null; }
-        S System.Linq.IQueryProvider.Execute<S>(System.Linq.Expressions.Expression expression) { throw null; }
-        public override string ToString() { throw null; }
+        TElement System.Linq.IQueryProvider.Execute<TElement>(System.Linq.Expressions.Expression expression) { throw null; }
+        public override string? ToString() { throw null; }
     }
     public static partial class Queryable
```

Comment:
It looks like you didn't annotate Queryable... I'd expect most of the annotations/attributes we have in System.Linq's Enumerable to also show up here.  For example, `[return: MaybeNull]` on the XxOrDefault methods that could end up returning null for reference types.  Please go through the LINQ surface area that's been annotated and confirm all annotations and attributes there are appropriately mirrored here.

Reply:
I'm aware. Holding off this PR until Expressions are annotated.

Reply:
Cool. Thanks.

Reply:
(Sorry, just saw your PR comment to this affect earlier.)